src/collectHeatmapClicks.js

```diff
@@ -113,7 +131,14 @@ export default (subscribe, { interval = 100 }) => {
   };
 
   injectSevenTagConfiguration();
-  const eventListener = throttle(listener);
-  // throttle to prevent click events spam
+
+  // prevent event loop after resume event click
+  const isEventReal = (event, callback) => {
+    if (event.screenX && event.screenX !== 0 && event.screenY && event.screenY !== 0) {
```

Comment:
You could use event.isTrusted but mind that:

![image](https://user-images.githubusercontent.com/12684955/116658723-b612c400-a990-11eb-881f-a1cd7ed8da89.png)

Reply:
Know it. I though to check that first (for browsers that now IE) and than make that as a fallback check for IE just in case but I think that current solution will cover all browsers as well.